update_employees.py

```diff
@@ -417,26 +418,23 @@ def main():
         name_field,
         result,
     )
-    cleaned_payload = remove_empty_emails(payload, email_field, name_field)
 
-    logging.info(f"{len(cleaned_payload)} total records to process in Knack.")
+    logging.info(f"{len(payload)} total records to process in Knack.")
 
+    result["additions"] = [record for record in payload if not record.get("id")]
+    result["updates"] = [record for record in payload if record.get("id")]
     result["errors"] = []
 
-    for record in cleaned_payload:
+    for record in payload:
         method = "update" if record.get("id") else "create"
         try:
             logging.info(f"{method} {record[email_field]['email']}")
             app.record(data=record, method=method, obj=knack_obj)
         except requests.HTTPError as e:
             if e.response.status_code == 400:
-                if record["field_230"] == "Crossing Guard":
```

Comment:
i was hitting an occasional type error here dealing with de-activated users. i think it's fine to log all this below with the rest of the errors